### PR TITLE
Add development chart

### PR DESCRIPTION
**Pull request by jacobsee**
First dev environment commit of the OMP apps

### CODE REVIEW

File: development/templates/buildconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      name: '{{ .Values.imageName }}:{{ .Values.imageTag }}'
+  postCommit: {}
+  resources: {}
+  runPolicy: Serial
+  source:
+    git:
+      ref: {{ .Values.git.ref }}
+      uri: {{ .Values.git.uri }}
+    type: Git
+    dockerfile: |
```
Comment (oybed) [inline development/templates/buildconfig.yaml]: It worries me a bit that we now have 2 different build "scripts" that can easily be out of sync. Couldn't we use something as simple as a shell script to keep it consistent between this and buildConfig and the GitHub actions?

Comment (jacobsee) [reply development/templates/buildconfig.yaml]: Having trouble visualizing a script for keeping a buildconfig in sync with the github action... Can you elaborate a bit on what that might look like?

Comment (oybed) [reply development/templates/buildconfig.yaml]: To stay consistent between how we build for developers, and how we build the "official" images, using the same script for the build steps involved seems important. GitHub actions can call a shell script that's part of the repo, e.g:
```
   run: |
     sh ./scripts/build.sh
```
In that way, this BuildConfig using in OpenShift could also call this `scripts/build.sh` and hence be consistent. 

Or, potentially even better yet, just add a `Dockerfile` to this repo, change the GitHub Actions to use it \[1\] and change the BuildConfig to a `source` strategy that will use the same Dockerfile by pointing to this repo. 

\[1\]: https://help.github.com/en/actions/building-actions/creating-a-docker-container-action

Comment (oybed) [reply development/templates/buildconfig.yaml]: Will be done in follow-up PRs

**Pull request merged**